.github/workflows/build-distribute.yml

```
@@ -0,0 +1,225 @@
+          export FIREBASE_APP_ID=${{ secrets.FIREBASE_APP_ID }}
+
+          echo "Capitalized Variant: $capitalizedVariant"
+          if [ "${{ inputs.build_type }}" == "debug" ]; then
+            bundle exec fastlane build_and_distribute_debug variant:"$capitalizedVariant"
+          else
+            bundle exec fastlane build_and_distribute_release variant:"$capitalizedVariant"
+          fi
+
+
+
+  upload_artifacts:
```

actionlint (1.7.4)

job "upload_artifacts" needs job "build" which does not exist in this workflow

(job-needs)